### PR TITLE
add organization folder support

### PR DESCRIPTION
**Pull request by coreyjonoliver**

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/pipeline/multibranch/defaults/PipelineMultiBranchDefaultsProjectFactory.java [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+     * @param useSandbox Set true to enable Groovy sandbox or false to run in
+     *                   Jenkins master runtime.
+     */
+    @DataBoundSetter
+    public void setUseSandbox(boolean useSandbox) {
+        this.useSandbox = useSandbox;
+    }
+
+    /**
+     * Get the current setting for whether or not to use a groovy sandbox.
```
Comment (NickolasHKraus) [inline src/main/java/org/jenkinsci/plugins/pipeline/multibranch/defaults/PipelineMultiBranchDefaultsProjectFactory.java]: nit: **G**roovy (to match method comment for `setUseSandbox`)

Comment (NickolasHKraus) [reply src/main/java/org/jenkinsci/plugins/pipeline/multibranch/defaults/PipelineMultiBranchDefaultsProjectFactory.java]: Same for `@return`

**Comment by coreyjonoliver** [comment]
@samrocketman Do you mind taking the opportunity to review this when you have the chance? Much appreciated!

**Comment by shamil** [comment]
Really useful feature, hope will be merged soon!

**Comment by samrocketman** [comment]
Hello, I was moving cross country for a couple months and was severely AFK from volunteer work.  I'm starting to get back into the groove of things and will take a look at this.

File: pom.xml [hunk 2]
```diff
@@ -37,7 +37,7 @@ THE SOFTWARE.
         <java.level>8</java.level>
     </properties>
     <artifactId>pipeline-multibranch-defaults</artifactId>
-    <version>2.1-SNAPSHOT</version>
+    <version>2.2-SNAPSHOT</version>
```
Comment (samrocketman) [inline pom.xml]: No need to bump the snapshot version.  It will be auto-bumped upon next plugin release.

Comment (coreyjonoliver) [reply pom.xml]: Fixed.

**Comment by coreyjonoliver** [comment]
@samrocketman Split my work into two commits, as suggested.

File: pom.xml (hunk 2 above)
Comment (samrocketman) [reply pom.xml]: Thanks!  I’ve been away a few months while moving cross country.  I am back now

Comment (samrocketman) [reply pom.xml]: The two commits is much more clear and makes it easier to review.  Thanks for taking the time to do it.

**Comment by NickolasHKraus** [comment]
@samrocketman Mind giving this a review when you get a chance?

**Comment by caitlin615** [comment]
Really looking forward to this being released! Any idea of the timeline?

**Comment by coreyjonoliver** [comment]
Just a friendly bump. @samrocketman do you mind taking another look at this?

**Comment by coreyjonoliver** [comment]
@samrocketman sorry to sound like a broken record, but could you give this another review?

**Comment by coreyjonoliver** [comment]
Great news! Thanks for taking the time to look it over @samrocketman.

**Comment by samrocketman** [comment]
I'll be able to finish this up today.  Apologies for the delay.  I went on a bit of a hiatus from FOSS development because moving cross country kind of drained my motivation for a lot of things.  Switching social circles and other non-visible things.  I really appreciate your patience with me as a maintainer.

**Comment by samrocketman** [comment]
Successfully tested.  This feature has been released and should be available in the update center within about 8 hours.  If you want to try it now you can download https://repo.jenkins-ci.org/releases/org/jenkins-ci/plugins/pipeline-multibranch-defaults/2.1/pipeline-multibranch-defaults-2.1.hpi

**Comment by casz** [comment]
Thanks @coreyjonoliver for your contribution!
This is super awesome exactly what I was looking for!
😍

**Pull request merged**